VacaYAY/DataAccesLayer/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DataAccesLayer.Configuration;
+using DataAccesLayer.Entities;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+
+namespace DataAccesLayer.Data
+{
+    public class ApplicationDbContext : IdentityDbContext<Employee, IdentityRole<int>, int>
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options) { }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            builder.ApplyConfiguration(new EmployeeConfiguration());
```

Comment:
Make your life easier 😄 
Use this: https://learn.microsoft.com/en-us/ef/core/modeling/#applying-all-configurations-in-an-assembly